www/js/model.js

```diff
@@ -268,11 +272,32 @@ define([
 					}, this);
 				}
 			},
+			updateLinks: function(/*string*/ id){
+				var nodes = this.getNodes();
+				var removeId = id + "_initial";
```

Comment:
What is the usage of this function? Would it be possible to add some comments to this one. I think this function already existed. From the looks of it, I am not completely sure.